js/main.js

```diff
@@ -1,30 +1,129 @@
+
+// Поле avatar — это строка, значение которой формируется по правилу img/avatar-{{случайное число от 1 до 6}}.svg. Аватарки подготовлены в директории img.
 
-getRandomInteger(0, 49);
+// Для формирования текста комментария — message — вам необходимо взять одно или два случайных предложения из представленных ниже:
 
+// Всё отлично!
+// В целом всё неплохо. Но не всё.
+// Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.
+// Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.
+// Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.
+// Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!
+// Имена авторов также должны быть случайными. Набор имён для комментаторов составьте сами. Подставляйте случайное имя в поле name.
 
-//Функция для проверки максимальной длины комментария.
+const PICTURE_QTY = 25;
 
-const checkLength = function (text, maxLength) {
-  if (text.length > maxLength) {
-    console.log(`Длина комментария составляет ${text.length}, что больше допустимых ${maxLength} символов`);
-    return false;
+const descriptionArray = [
+  'Кекс на море',
+  'Рабочие будни',
+  'Поездка в Испанию',
+  'Отдых у бабушки',
+  'На тренировке',
+  'Кошачьи лапки',
+  'Прыжки от стены',
+  'Погром в офисе',
+  'Украденные тапки',
+  'Хозяин здесь - я',
+  'Дома, с любимой',
+  'Пишу для тебя ТЗ',
+  'Билеты в Сан-Франциско',
+  'Кекс на весах',
+  'После тяжелого дня',
+  'Едем шопиться!',
+  'Летучка в офисе',
+  'Обеденная сиеста',
+  'Полет на дельтаплане',
+  'Воздушный поцелуй',
+  'Читаю последние новости',
+  'Селфи на фоне Парижа',
+  'Пробую новое вино',
+  'Выбираю подарки на 8 марта',
+  'Я - красивый!'
+];
+
+const messageArray = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+];
+
+const namesArray = [
+  'Иван',
+  'Хуан',
+  'Мария',
+  'Кристофф',
+  'Виктор',
+  'Юлия',
+  'Себастьян',
+  'Ахмед'
+];
+
+const idArray = [];
+let result;
+function getRandomInteger (a, b, noRepeat, array) {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  result = Math.floor(Math.random() * (upper - lower + 1) + lower);
+  if (noRepeat) {
+    if (array.includes(result)) {return getRandomInteger(a, b, noRepeat, array);}
+    else {
+      array.push(result);
+      return result;
+    }
   }
```

Comment:
```suggestion
function getRandomInteger (a, b) {
  const result = Math.floor(Math.random() * (upper - lower + 1) + lower);
  
  return result;
```
вот тут как-то сложно. функция которая отвечает за генерацию случайного числа, ничего не должна делать больше, чем просто генерить случайное число